final_task/rss_reader/myargparse.py

```diff
@@ -0,0 +1,20 @@
+import argparse
+
+parser = argparse.ArgumentParser(description='Getting info from sites')
```

Comment:
Entire logic of this module should placed in a function.

Reply:
Ok